src/tactic/command/login_group_trigger.py

```diff
@@ -46,14 +47,36 @@ def execute(my):
 
         login_group = login_group_name.lower()
         login_group = login_group.replace(" ", "_")
```

Comment:
what if they use ' or ! or other symbols in their login_group. I would use a re.sub() function to replace a few more symbols or all non-word character with _.  What if they use non ASCII characters.. would lower() work with that?